general/development/tools/behat.md

```diff
@@ -0,0 +1,721 @@
+   ```
+
+2. Open your Moodle installation's `config.php` in your preferred editor, and require the tool's `init.php`:
+
+   ```
+      require_once('/path/to/your/git/dir/moodle-browser-config/init.php');
+   ```
+
+##### Provided profiles
+
+The full list of profiles which are included with `moodle-browser-config` are provided in its [own documentation](https://github.com/andrewnicols/moodle-browser-config).
+
+The following is a summary of the profiles that most users may be interested in.
+
+<!-- cspell:ignore Browserstack, Saucelabs, headlessfirefox, geckodriver, Geckodriver, chromedriver, Chromedriver -->
+<!-- cspell:ignore headlessgeckodriver, headlesschrome, headlesschromedriver, headlessedge, Edgedriver, edgedriver -->
+<!-- cspell:ignore headlessedgedriver, MYTHEME, rehomed -->
+
+You can also provide your own custom profiles, including for remote services such as Browserstack, and Saucelabs, as
+well as for other browsers supporting the W3C Webdriver specification.
+
+Please note that `Safari` and `Safaridriver` are not currently supported as they do not meet the W3C WebDriver specification.
+
+| Profile name | Description | Uses Selenium? | Displays GUI? |
+| --- | --- | --- | --- |
+| firefox | Use Firefox via Selenium | Yes | Yes |
+| headlessfirefox | Use Firefox via Selenium, without displaying the GUI | Yes | No |
+| geckodriver | Use Firefox with Geckodriver directly | No | Yes |
+| headlessgeckodriver | Use Firefox with Geckodriver directly, without displaying the GUI | No | No |
+| chrome | Use Chrome via Selenium | Yes | Yes |
+| headlesschrome | Use Chrome via Selenium, without displaying the GUI | Yes | No |
+| chromedriver | Use Chrome with Chromedriver directly | No | Yes |
+| headlesschromedriver | Use Chrome with Chromedriver directly, without displaying the GUI | No | No |
+| edge | Use Edge via Selenium | Yes | Yes |
+| headlessedge | Use Edge via Selenium, without displaying the GUI | Yes | No |
+| edgedriver | Use Edge with Edgedriver directly | No | Yes |
+| headlessedgedriver | Use Edge with Edgedriver directly, without displaying the GUI | No | No |
+
+#### chromedriver-wrapper
+
+When using Google Chrome, you must use the correct version of the `chromedriver` browser driver for the version of Chrome that you use.
+
+Since Google Chrome automatically updates on a regular basis, you will need to regularly upgrade your driver to match the version of Chrome that you are using.
+
+To make this easier, a `chromedriver-wrapper` utility has been written. This inspects the version of Chrome that is in your path, and downloads the correct version of `chromedriver` for that version, before starting it.
+
+Installation instructions can be found at [https://github.com/andrewnicols/chromedriver-wrapper](https://github.com/andrewnicols/chromedriver-wrapper).
+
+## Getting started (basic)
+
+This is a quick walk through to get Behat running for the first time.
+
+### Setting up
```

Comment:
I would suggest slightly reorganising this section in order to make it easier to navigate using the page index:
- Getting started
  - Setting up Selenium
  - Setting up your browsers
  - Setting up Moodle
  - Configure Behat for Moodle --> Initialise Behat
  - Run Behat tests
- Advanced testing